src/tests/end-to-end/tests/details-view/__snapshots__/preview-features.test.ts.snap

```diff
@@ -58,7 +58,7 @@ exports[`Details View -> Preview Features Panel should match content in snapshot
         >
           <p
             aria-level="2"
-            class="ms-Panel-headerText header-text headerText-000"
+            class="ms-Panel-headerText header-text--000nPhV headerText-000"
```

Comment:
This edit suggests something is wrong with the interaction between the E2Es' formatElementForSnapshot logic and the webpack settings for identifier suffixes; it's supposed to be normalizing this. I think we were expecting the webpack settings to be adding a suffix of form `--09f` (ie, 3 characters of hash only).

Don't need to fix it for this PR, but we should look into it separately.

Reply:
Is this related to this line on `webpack.config.js`:
```javascript
localIdentName: '[local]--[hash:base64:5]',
```
If that's so, doesn't the `:5` suggest we'll get 5 characters not 3 as you suggest?